### PR TITLE
Big Iron gets some love

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
Big Iron gets something to make it unique.

## Why It's Good For The Game
Big Iron unlike its counterpart Magic Bullet didn't get anything to work with, you didn't get any effect other than the armor and drip. 
This PR aims to change that by instead of giving it the attack speed of the magic bullet, it gives it **BIG DAMAGE** to be the **BIG IRON** it always was meant to be with a huge **2.5** damage multiplier. 

Do you think that's too much? You know that magic bullet armor doubles its attack speed from a delay of 30 to 15 right? I think this makes it just strong enough to not break any walls unlike some others _*cough *cough wounded courage *cough *cough_

## Changelog
:cl:
add: Added passive to Big Iron
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 1]
```diff
@@ -62,7 +64,7 @@
 	inhand_icon_state = "magic_bullet"
 	special = "This weapon fires extremely slowly. \
 		This weapon pierces all targets. \
-		This weapon fires significantly faster wearing the matching armor"
+		This weapon becomes stronger when wearing the matching armor"
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/ego_gun/waw.dm:67]: Put this on the Big Iron armor.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 2]
```diff
@@ -76,10 +78,17 @@
...
 	var/mob/living/carbon/human/myman = user
 	var/obj/item/clothing/suit/armor/ego_gear/magicbullet/Y = myman.get_item_by_slot(ITEM_SLOT_OCLOTHING)
+	var/obj/item/clothing/suit/armor/ego_gear/realization/bigiron/Z = myman.get_item_by_slot(ITEM_SLOT_OCLOTHING)
 	if(istype(Y))
 		fire_delay = 15
+	if(istype(Z))
+		projectile_damage_multiplier = 2.5
+	else
+		projectile_damage_multiplier = 1
+	return
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/ego_gun/waw.dm:88]: Remove this return, this stops it from firing entirely

Comment (BlueHNT) [reply code/modules/projectiles/guns/ego_gun/waw.dm:88]: It worked fine in testing, but sure!

File: code/modules/projectiles/guns/ego_gun/waw.dm (hunk 1 above)
Comment (BlueHNT) [reply code/modules/projectiles/guns/ego_gun/waw.dm:67]: image

**Review by Kitsunemitsu — APPROVED** [review]
Approved

**Pull request merged**